boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_iron_mcuboot_b0.dts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54h20dk_nrf54h20_cpuapp.dts"
+#include "nrf54h20dk_nrf54h20-ipc_conf_iron.dtsi"
+#include "nrf54h20dk_nrf54h20-memory_map_iron_mcuboot_b0.dtsi"
+
+/delete-node/ &cpusec_cpurad_ipc;
+
+/ {
+	chosen {
+		zephyr,code-partition = &slot0_partition;
+		zephyr,uart-mcumgr = &uart136;
+	};
+};
+
+&cpusec_bellboard {
+	status = "okay";
+};
+
+&cpusec_cpuapp_ipc {
+	mbox-names = "tx", "rx";
+	status = "okay";
+};
+
+b0_partition: &cpuapp_b0_partition {
+	label = "b0";
+};
+
+provision_partition: &cpuapp_provision_partition {
+	label = "provision";
+};
+
+boot_partition: &cpuapp_boot_partition {
+	label = "mcuboot";
+};
+
+boot_partition_slot1: &cpuapp_boot_slot1_partition {
+	label = "mcuboot-image-1";
+};
```

Comment:
as per sdk-nrf PR, needs proper names

Reply:
No longer applicable, moved board definition to sdk-nrf